UserTools/Configuration/Configuration.cpp

```diff
@@ -40,7 +42,7 @@ bool Configuration::Execute(){
   
   if(m_data->load_config){
 
-    while(!LoadConfig());
+    if(!LoadConfig()) return false;
```

Comment:
needs to be while becuase we want it to keep quirying the db until it gets it not fail and give up.

Reply:
we saw in WCTE this filled the database with 900 errors. :grimacing: